benchexec/util.py

```diff
@@ -762,3 +762,22 @@ def check_msr():
         if all(os.access("/dev/cpu/{}/msr".format(cpu), os.W_OK) for cpu in cpu_dirs):
             res["write"] = True
     return res
+
+
+def get_data_model_from_task(task, data_models):
```

Comment:
That's what I was wondering too. I thought adding a separate `svcomp_utils` module in `tools` a bit of overkill.

I can add a `svcomp_util` in the `tools` package

Reply:
Maybe it makes more sense to add an abstract class for SV-COMP tools?

Reply:
We can add an abstract class named `CompetitionTool` because this change is relevant also for `TestComp`.

This change is is relevant for tools dealing with the language `C`, not `Java`. 
Shall we call it `CompetitionToolsForC`?

Where do you want to place this class? Separate file or `template.py`?

Reply:
Hm, `CompetitionTool` or something similar would sound like this affects all kinds of competitions. In fact, its content is not even related to SV-COMP, but to SV-Benchmarks (where the `data_model` key is [defined](https://github.com/sosy-lab/sv-benchmarks#task-definitions)). So a name pointing in that direction might make sense, or something related to `C`.

We certainly should not place this class in `template.py` to make it clear it is not required by BenchExec for and tool-info modules and that it is not part of the public API. Probably we should even clearly state that out-of-tree modules should not use it and that it may change at any point in time, otherwise we would also have to version it like we do with `BaseTool2`.

Reply:
IMO it is much simpler to put it in a util file. I will do that for now.

Reply:
@PhilippWendler do you agree with this?

Reply:
I would say it should have a less generic name. `toolinfo_util` sounds like an official file with officially supported API addons.

Reply:
do you have a suggestion?

I would like to reach an agreement before making the changes.

Reply:
`sv_benchmarks_util`?

Reply:
fixed in 14cd4fc0477b9443c6b58a4824273510afdf076e